src/Ball.js

```diff
@@ -109,15 +114,18 @@ export class Ball {
             this.state = STATES.incubating
             RUN.results[STATES.incubating]++
             RUN.results[STATES.well]--
-            if (this.hasAppInstalled && otherBall.hasAppInstalled) {
+
+            if (this.hasTheAppInformedOfAContactWithAnInfectedPerson(otherBall)) {
```

Comment:
The previous code was easier to be read (and understood) than this very long function name.
I would suggest a shorter name for it, e.g. `awareToBeInfectedBy`

Reply:
Yes, although I advise to always use the prefix is/has when a boolean is expected. isAwareToBeInfected will do

Reply:
Change mved forward to next PR.